### ISSUE TITLE
Future Warning from sklearn for KernelExplainer concerning LassoLarsIC

### ISSUE THREAD

**Issue by c-feldmann** [shap/shap#2528]
Calling the method `shap_values` from the  `KernelExplainer` gives the following warning:
```
The default of 'normalize' will be set to False in version 1.2 and deprecated in version 1.4.

If you wish to scale the data, use Pipeline with a StandardScaler in a preprocessing stage. To reproduce the previous behavior:

from sklearn.pipeline import make_pipeline

model = make_pipeline(StandardScaler(with_mean=False), LassoLarsIC())

If you wish to pass a sample_weight parameter, you need to pass it as a fit parameter to each step of the pipeline as follows:

kwargs = {s[0] + '__sample_weight': sample_weight for s in model.steps}
model.fit(X, y, **kwargs)

Set parameter alpha to: original_alpha * np.sqrt(n_samples). 
```
Versions:
python: 3.9.12 (conda)
sklearn: 1.0.1
shap: 0.40.0
OS: Ubuntu 18.04.1 LTS

**Comment by Aboubacar-cmr** [comment]
Oui du coup quelqu'un sais c'est quoi la solution?

**Comment by c-feldmann** [comment]
@Aboubacar-cmr is this translation correct?
"Yes, so does anyone know what the solution is?"
Then the answer is, at least from my site: No, i simply silenced the warning:

```python
with warnings.catch_warnings():
    warnings.filterwarnings("ignore")
    shap_values = model_explainer.shap_values(X_train)
```

**Comment by Aboubacar-cmr** [comment]
@c-feldmann merci beaucoup.
Il n'existe pas d'autre solution sans pour au tant ignoré l'erreur?

**Comment by c-feldmann** [comment]
@Aboubacar-cmr 
Could you phrase your questions in English, please?
Assumed translation: "Is there no other solution without ignoring the error?"

As I stated before, no, there is no solution I'm aware of.

**Comment by Aboubacar-cmr** [comment]
Okay, Thanks @c-feldmann

**Comment by alvaromiguelo** [comment]
Define the parameter normalize to False and you should get rid of the warning.

**Comment by courtarro** [comment]
> Define the parameter normalize to False and you should get rid of the warning.

Where?
